### DISCUSSION TITLE
Can anyone give me a list of All rest api endpoints that supabase have ?

### DISCUSSION THREAD

**Discussion by ARayane24** [supabase/supabase#22069] [Questions]
Hello everyone,
I'm trying to make java lib to support supabase for that I need a list of rest APIs to make all http requests
and it's looks like supabase doc. Hasn't mentioned anything about, is it forbidden to have them ??
if not, i'll appreciate any kind of help

**Comment by GaryAustin1** [comment]
There is not a single list.
You can go to the various repositories and likely get the info.  So auth, storage, realtime, etc github repositories and postgrest.org.

**Reply by ARayane24** [reply]
hi @GaryAustin1 thank you,
I just thought that someone may have them, or at least tel me where are they .. Instead of looking for them in all git repos 😅
